augur/export_v2.py

```diff
@@ -877,6 +887,9 @@ def register_parser(parent_subparsers):
                                  help="delimiters to accept when reading a metadata file. Only one delimiter will be inferred.")
     optional_inputs.add_argument('--metadata-id-columns', default=DEFAULT_ID_COLUMNS, nargs="+",
                                  help="names of possible metadata columns containing identifier information, ordered by priority. Only one ID column will be inferred.")
+    optional_inputs.add_argument('--metadata-columns', nargs="+",
+                                 help="Metadata columns to export in addition to columns provided by --color-by-metadata or --auspice-config. " +
```

Comment:
Is this just metadata or could we also export node-data-JSON derived data this way? I would favor making it draw from both data sources.

Reply:
This only works for TSV metadata. I purposely excluded the node-data JSON derived data in this PR to lower the complexity and reduce confusion. 

From the TSV metadata side, this is _adding_ columns that would have not been originally exported. While on the node-data side, all fields are automatically exported but we have to remove them from the coloring options. In my mind, these are different enough functions that I would make them separate CLI options.